.github/workflows/ci.yaml

```
@@ -5,63 +5,88 @@ on:
-          sudo mv hadolint /usr/local/bin/hadolint
+        run: bundle exec rubocop || true 
 
-      # Verify Hadolint installation
-      - name: Verify Hadolint installation
-        run: |
-          hadolint --version
+  # Job 3: Lint Dockerfiles
+  lint-docker:
+    name: Dockerfile Linting
+    runs-on: ubuntu-latest
+    needs: [] # Runs in parallel with 'test' and 'lint-ruby'
```

actionlint (1.7.4)

"needs" section should not be empty

(syntax-check)